dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -16,22 +16,27 @@
 # under the License.
 from __future__ import annotations
 
+import logging
 import os
 import re
 import shlex
 import shutil
 import sys
 import textwrap
 import time
+from collections import defaultdict
 from copy import deepcopy
 from datetime import datetime
 from pathlib import Path
 from subprocess import DEVNULL
 from typing import IO, Any, Generator, NamedTuple
 
 import click
+import semver
+from github import Github, UnknownObjectException
 from rich.progress import Progress
 from rich.syntax import Syntax
+from tests.test_utils.providers import get_provider_min_airflow_version
```

Comment:
No imports from airflow in `breeze` sorry :) . Breeze venv is (and should be) separate from Airflow :).

You should use `get_min_airflow_version` from breeze's `utils.packages`

Reply:
BTW. Breeze has quite comprehensive and "unitized" set of utils - each group of utils has it's own module in utils. I am from the school where those util classes should be split to many independend small set of closely related utils that can be imported and used independently from each other (avoid all kinds of circular dependencies and diamond relationships.

Reply:
done